doc/users/whats_new.rst

```diff
@@ -220,136 +373,21 @@ from :func:`~matplotlib.pyplot.figure`, it can also be used there::
-is greater than the ``path.simplify_threshold`` parameter. Thus, higher
-values of ``path.simplify_threshold`` result in quicker rendering times.
-If you are plotting just to explore data and not for publication quality,
-pixel perfect plots, then a value of ``1.0`` can be safely used. If you
-want to make sure your plot reflects your data *exactly*, then you should
-set ``path.simplify`` to false and/or ``path.simplify_threshold`` to ``0``.
-Matplotlib currently defaults to a conservative value of ``1/9``, smaller
-values are unlikely to cause any visible differences in your plots.
-
-
-Added `matplotlib.ticker.PercentFormatter`
-------------------------------------------
-
-The new `~matplotlib.ticker.PercentFormatter` formatter has some nice features
-like being able to convert from arbitrary data scales to percents, a
-customizable percent symbol and either automatic or manual control over the
-decimal points.
+:class:`~matplotlib.ticker.LogFormatterMathtext` now includes the
+option to specify a minimum value exponent to format as a scalar
+(ie. 0.001 instead of 10\ :sup:`-3`).
 
 
 New quiverkey angle keyword argument
 ------------------------------------
 
-Plotting a :func:`~matplotlib.axes.Axes.quiverkey`  now admits the ``angle``
-keyword argument, which sets the angle at which to draw the key arrow.
-
-
-Reproducible PS, PDF and SVG output
------------------------------------
-
-The ``SOURCE_DATE_EPOCH`` environment variable can now be used to set
-the timestamp value in the PS and PDF outputs. See
-https://reproducible-builds.org/specs/source-date-epoch/
-
-Alternatively, calling ``savefig`` with ``metadata={'creationDate': None}``
-will omit the timestamp altogether.
-
-The reproducibility of the output from the PS and PDF backends has so
-far been tested using various plot elements but only default values of
-options such as ``{ps,pdf}.fonttype`` that can affect the output at a
-low level, and not with the mathtext or usetex features. When
-matplotlib calls external tools (such as PS distillers or LaTeX) their
-versions need to be kept constant for reproducibility, and they may
-add sources of nondeterminism outside the control of matplotlib.
-
-For SVG output, the ``svg.hashsalt`` rc parameter has been added in an
-earlier release. This parameter changes some random identifiers in the
-SVG file to be deterministic. The downside of this setting is that if
-more than one file is generated using deterministic identifiers
-and they end up as parts of one larger document, the identifiers can
-collide and cause the different parts to affect each other.
-
-These features are now enabled in the tests for the PDF and SVG
-backends, so most test output files (but not all of them) are now
-deterministic.
-
+Plotting a :func:`~matplotlib.axes.Axes.quiverkey` now admits the
+``angle`` keyword argument, which sets the angle at which to draw the
+key arrow.
 
 Colormap reversed method
 ------------------------
```

Comment:
just below: `reversed` should not be prefixed with a tilde, otherwise the rendered docs look just weird.